exec/java-exec/src/main/java/org/apache/drill/exec/store/pcapng/schema/Schema.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.store.pcapng.schema;
+
+import org.apache.drill.exec.store.pcapng.schema.columns.Column;
+import org.apache.drill.exec.store.pcapng.schema.columns.DstIpImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.DstMacImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.DstPortImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.PacketDataImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.PacketLenImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.SrcIpImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.SrcMacImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.SrcPortImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpAckImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlags;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsAckImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsCwrImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsEceCongestionExperiencedImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsEceEcnCapableImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsEceImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsFinImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsNsImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsPshImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsRstImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsSynImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpFlagsUrgIml;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpParsedFlags;
+import org.apache.drill.exec.store.pcapng.schema.columns.TcpSessionImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TimestampImpl;
+import org.apache.drill.exec.store.pcapng.schema.columns.TypeImpl;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Schema {
```

Comment:
This `Schema` design is very nice: simple, easy to understand. Let's encourage others to use this kind of design for future readers.

Reply:
Thanks.